.eslintrc.js

```diff
@@ -106,7 +106,7 @@ module.exports = {
         "no-magic-numbers": [
             "error",
             {
-                ignore: [-1, 0, 1, 100],
+                ignore: [-1, 0, 1, 2, 100],
```

Comment:
This addition probably makes sense, though in that case, please remove 
```
/* eslint-disable-next-line no-magic-numbers */
```
where the magic number 2 is currently being used in the project, since the line is not necessary anymore. Alternatively, ignore the 2 in your new file and remove it from the ignore list.

Reply:
Makes sense! Since I changed the config when reworking this, I also removed the unnecessary exemptions now.